docs/changes.rst

```diff
@@ -117,6 +123,13 @@ Enhancements
 * Added a ``host`` property to ``WBEMConnection`` which contains the host:port
   component of the WBEM server's URL.  This helps addressing issue #349.
 
+* Made sure that ``repr()`` on CIM objects produces a reliable order of
+  items such as properties, qualifiers, methods, parameters, scopes, by
+  ordering them by their names. This makes debugging using ``repr()`´ easier
```

Comment:
repr() in the line above has a closing forward quote where it should have a back quote.

Reply:
Fixed